extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -66,7 +66,7 @@ require([
 
 		sourcePoint.initDetection();
 
-		if (context.opts.pageFairDetection) {
+		if (pageFair.canBeInitialized(context)) {
 			pageFair.initDetection(context);
```

Comment:
and you are doing the check one more time in Mercury. Isn't the check inside initDetection sufficient?